### DISCUSSION TITLE
Using inertiajs in my own custom PHP code

### DISCUSSION THREAD

**Discussion by jeankyle** [inertiajs/inertia#1123] [Help]
Hello, I have a basic working MVC structure, and I am wondering if there is a way to integrate inertiajs with. All tutorials I find are using somethong else. Vue, React, Laravel, etc. 

How do I get to use it if I have my own custom code? I am not using any frameworks, be it in the client side or the server side

**Comment by WalrusSoup** [comment]
I would suggest looking at [inertia-wordpress](https://github.com/boxybird/inertia-wordpress) for a starting point at rolling your own _frameworkless_ adapter.
